shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -64,6 +64,10 @@
     // PREFIX can be loaded from config when config becomes dependency.
     public static final String PREFIX = "pravega" + ".";
 
+    // Metrics for all services
+    public static final String LOG_ERRORS = PREFIX + "logged_errors";     // Counter
```

Comment:
Why not just name as ERROR_COUNTER, WARN_COUNTER? LOG_ERRORS sounds logs more than counter.

Reply:
That would be inconsistent with the naming of the other metrics defined in this class.

Plus, I want the name "Log" in it so that I can make it clear that the source of the information is the logs, and not something else.